internal/jspackages/js_package_dep.go

```diff
@@ -26,6 +27,10 @@ func FindPackageDependencies(wd string) (depbot.Dependencies, error) {
 	var hasPackageLockDeps bool
 
 	filepath.WalkDir(wd, func(path string, d fs.DirEntry, err error) error {
+		if strings.Contains(path, "node_modules") {
```

Comment:
This one should use `https://pkg.go.dev/path/filepath#SplitList` and then check wether in the list there is a `node_modules` element. Otherwise cases like `/mynode_modules/` or `node_modules_should_be_included` will be ignored.

Reply:
I think for this one ill use the `strings.Split(path, "/")` because the SplitList uses as a separator `:` instead of `/`

Reply:
In that case, we cannot hardcode "/" because Windows uses "\". We should use `filepath.Separator`.

Reply:
Yes that makes sense.
Just updated